### DISCUSSION TITLE
page doesnt publish

### DISCUSSION THREAD

**Discussion by hasanmd91** [community/community#30857] [Other Feature Feedback, Questions, & Ideas]
I am, trying to publish my project but it's not working! any idea what's happening wrong?
https://hasanmd91.github.io/html_css_vera/

repo : https://github.com/hasanmd91/html_css_vera/tree/master/week-3/signup-form

**Comment by airtower-luna** [comment]
It seems to be deployed just fine, there just isn't any `index.html` at the top level of your repository, so the README gets turned into one. If you add the full path your form is there: https://hasanmd91.github.io/html_css_vera/week-3/signup-form/

If you don't want to write a full top-level page, the simplest thing would be to add links to the exercise subdirectories to your README. :wink:
